src/@next/components/molecules/FormFooter/FormFooter.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import { Button, ButtonLink } from "@components/atoms";
+import * as S from "./styles";
+import { IButtonProps, IProps } from "./types";
+
+const getBtnAction = (btn: IButtonProps) =>
+  btn.action && { onClick: btn.action };
+
+const renderCancelBtn = (cancelBtn?: IButtonProps) =>
+  cancelBtn && (
+    <ButtonLink {...getBtnAction(cancelBtn)} type="button" color="secondary">
```

Comment:
ButtonLink doesn't seem to have any hover/active styles - is this how it's supposed to be?

Reply:
it should have hover state...i'll fix it soon